src/vm/exceptionhandling.h

```diff
@@ -389,15 +389,15 @@ class ExceptionTracker
 
     bool IsStackOverflowException();
 
-#ifdef FEATURE_PAL
+#if defined(FEATURE_PAL) && !defined(CROSS_COMPILE)
```

Comment:
So why are you excluding the function? Is there any build warning related to its presence?